tools/STOC/STOCeps2en.xml

```diff
@@ -0,0 +1,135 @@
+<tool id="stoc2" name="STOCeps2" version="1.0">
+    
+    <description>Annual variation and global trend by specialist groups from STOCeps (simple spot sampling).</description>
+    
+    <requirements>
+        <requirement type="package">r</requirement>
+        <requirement type="package">r-r.methodss3 </requirement>
+        <requirement type="package">r-arm</requirement>
+    </requirements>
+
+    <stdio>
+        <exit_code range="1:" />
+    </stdio>
+    
+    <command>
+        <![CDATA[
+        $__tool_directory__/STOCeps2.R ${variationsAnnuelles} ${variationsGlobales} $__tool_directory__/Librairie ;
+        mv tendancesAnnuellesGroupes.tsv output1 ;
+        mv tendancesGlobalesGroupes.tsv output2 ;
```

Comment:
instead of move, just use from_work_dir https://docs.galaxyproject.org/en/latest/dev/schema.html#id63

Reply:
oh, I see you are using it, then the MV can be removed.